test/e2e/common_test.go

```diff
@@ -264,6 +264,10 @@ func PodmanTestCreateUtil(tempDir string, remote bool) *PodmanTestIntegration {
 	if rootless.IsRootless() {
 		storageFs = ROOTLESS_STORAGE_FS
 	}
+	//Need to keep storage driver consistent between STORAGE_OPTIONS and STORAGE_FS
```

Comment:
I don't understand this comment. Is this saying, "this section of code ensures that those two envariables are consistent"? Of is it saying, "STORAGE_FS must match STORAGE_OPTIONS, and it is up to the person invoking the test to guarantee that this is so"? (`test/e2e/config_*.go imply this). Or something else?

Under what circumstances will one be set without the other, or in conflict with each other? If they must always be set in agreement with each other, then that is evil, and there should be one and only one environment variable that controls both settings.

I have no idea what this does, and don't even know how to find out, and there is no documentation that I can find; so my comments may not be relevant. Either way, though, this comment is unclear.

Reply:
> I don't understand this comment. Is this saying, "this section of code ensures that those two envariables are consistent"? Of is it saying, "STORAGE_FS must match STORAGE_OPTIONS, and it is up to the person invoking the test to guarantee that this is so"? (`test/e2e/config_*.go imply this). Or something else?
> Under what circumstances will one be set without the other, or in conflict with each other? If they must always be set in agreement with each other, then that is evil, and there should be one and only one environment variable that controls both settings.

The value of STORAGE_FS is a hard code in test/e2e/config_amd64.go, we need to run tests to cover different scenario like 'overlay' storage driver on x86_64 platform, at present, we can set STORAGE_OPTIONS="--storage-driver overlay" to run e2e tests, but as I said,  the storage driver is unmatched between STORAGE_OPTIONS(overlay) and STORAGE_FS('vfs' hard code), the result is all of e2e tests are failure. 

It has two solutions to fix this at least:
1. export STORAGE_FS as environment variable, let testers can change hard code storage driver 
2. don't export STORAGE_FS, we need to add codes to keep storage driver consistent when storage driver is unmatched between default STORAGE_FS and STORAGE_OPTIONS

NOTE: for item 1, it's very flexible, but it needs to keep storage driver is consistent, we need to document this for tester, if you think it's evil, I can choose item 2, what do you think? or any good idea for this? thanks!

> 
> I have no idea what this does, and don't even know how to find out, and there is no documentation that I can find; so my comments may not be relevant. Either way, though, this comment is unclear.

Anyway, it should let tester to change hard code value of STORAGE_FS in test/e2e/config_amd64.go, otherwise, it will lost 'overlay' test scenario on x86_64 platform, if w/o this patch, we need to manually change STORAGE_FS: 'overlay' in est/e2e/config_amd64.go before testing, I think it's not what we want, right? Thanks!

Reply:
I'm not native speaker, if this comment "//Need to keep storage driver consistent between STORAGE_OPTIONS and STORAGE_FS" is not clear or incorrect, please help improve it, thanks a lot!